Index: print.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>name = input('What is your name?')\nprint('Hello, ', name + '!')\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- print.py	(revision c77024dbca1e54cc8ed10109c948bd5c8fb7331a)
+++ print.py	(revision )
@@ -1,2 +1,21 @@
-name = input('What is your name?')
-print('Hello, ', name + '!')
+def user():
+    login = input('login: ')
+    pswrd = input('password: ')
+    return login, pswrd
+
+
+
+def logon():
+    x = input('enter your login here: ')
+    y = input('enter your password here: ')
+    return x, y
+
+
+def compare():
+    if logon() == user():
+        print('welcome, ' + login.user)
+    else:
+        print('Wrong login or password')
+
+
+compare()
